<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>848</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tragwerksimulator</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="TrussWidget" name="myTrussWidget">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>500</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="Mouse_coordinate1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Mouse_coordinate2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Elastizitätsmodul:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLineEdit" name="E_modul_input"/>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="Emodul_aktiv">
                 <property name="text">
                  <string>aktivieren</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>linear</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="Emodul_speichern">
                 <property name="text">
                  <string>speichern</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Emodul_laden">
                 <property name="text">
                  <string>laden</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Knoten:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="X_input">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="Y_input"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Locator:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="Joint_locator"/>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="addJoint_Button">
                 <property name="text">
                  <string>hinzufügen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteJoint_Button">
                 <property name="text">
                  <string>löschen</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Lager:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="Lager">
               <item>
                <property name="text">
                 <string>mit Lager</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ohne Lager</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Kräfte:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>F&lt;sub&gt;x&lt;/sub&gt;:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Force_x"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>F&lt;sub&gt;y&lt;/sub&gt;:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Force_y"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Locator:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Force_Locator"/>
             </item>
             <item>
              <widget class="QPushButton" name="addForce_Button">
               <property name="text">
                <string>hinzufügen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteForce_Button">
               <property name="text">
                <string>löschen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Stäbe:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Dicke:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="thickness">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Locator 1:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="Edge_Joint1"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Locator 2:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="Edge_Joint2"/>
             </item>
             <item row="3" column="0" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="Knoten_verbinden">
                 <property name="text">
                  <string>verbinden</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteEdge_Button">
                 <property name="text">
                  <string>trennen</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>325</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionTragwerk_speichern"/>
    <addaction name="actionTragwerk_laden"/>
    <addaction name="actionneues_Tragwerk"/>
    <addaction name="actionAls_Bild_speichern"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <addaction name="actionAltes_Tragwerk_zeigen"/>
    <addaction name="actionNeues_Tragwerk_zeigen"/>
    <addaction name="actionVerschiebung_zeigen"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionSteuerung"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSimulieren"/>
  </widget>
  <action name="actionTragwerk_speichern">
   <property name="text">
    <string>Tragwerk speichern</string>
   </property>
  </action>
  <action name="actionTragwerk_laden">
   <property name="text">
    <string>Tragwerk laden</string>
   </property>
  </action>
  <action name="actionAltes_Tragwerk_zeigen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Altes Tragwerk zeigen</string>
   </property>
  </action>
  <action name="actionNeues_Tragwerk_zeigen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Neues Tragwerk zeigen</string>
   </property>
  </action>
  <action name="actionVerschiebung_zeigen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verschiebung zeigen</string>
   </property>
  </action>
  <action name="actionJointMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>JointMode</string>
   </property>
  </action>
  <action name="actionConnectionMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ConnectionMode</string>
   </property>
  </action>
  <action name="actionForceMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ForceMode</string>
   </property>
  </action>
  <action name="actionEraserMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EraserMode</string>
   </property>
  </action>
  <action name="actionSimulieren">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulieren</string>
   </property>
  </action>
  <action name="actionSteuerung">
   <property name="text">
    <string>Steuerung</string>
   </property>
  </action>
  <action name="actionneues_Tragwerk">
   <property name="text">
    <string>neues Tragwerk</string>
   </property>
  </action>
  <action name="actionAls_Bild_speichern">
   <property name="text">
    <string>Als Bild speichern</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrussWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">trusswidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
